test/unit/props.spec.js

```diff
@@ -17,20 +17,16 @@ describe('api/props', () => {
     elem = new (define(class extends Mixins.Props() {
       static get props () {
         return {
-          prop1: {
-            initial: 'test1'
-          },
-          prop2: {
-            initial: 'test2'
-          },
-          prop3: {
-            default: undefined
-          }
+          prop1: null,
+          prop2: null,
+          prop3: null
         };
       }
       constructor () {
         super();
         this._rendered = 0;
+        this.prop1 = 'test1';
```

Comment:
maybe can we one of the props move as property initialisers? I know that this is what it does under the hood ( babel / TS ) but just to have it in test as documentation reference would be nice.

```js
elem = new (define(class extends Mixins.Props() {

 prop2 = 'test2';

 constructor(){
   super();
   this._rendered = 0;
   this.prop1 = 'test1'
 }

})

```